src/pages/certificates/tests/certificates-page.spec.js

```diff
@@ -50,7 +50,6 @@ describe('Information about certificates page', () => {
       ...usersMock,
       loading: false
     }));
```

Comment:
In case that you didn`t use redux for users in this component his useSelector is redundant.

Reply:
ok,thanks)